templates/handlebars/_index.js

```diff
@@ -142,6 +142,10 @@ module.exports = function (dest, ctx) {
    */
   ctx.data.byGroupAndType = extras.byGroupAndType(ctx.data);<% } %>
 
+  // Avoid key collision with Handlebars
+  ctx._data = ctx.data;
```

Comment:
Still not 100% clear where data goes from an `Array` of `Objects` to `true`, but for the life of me I can't figure out what else could be the culprit besides Handlebars' Compiler.